python/pdstools/plots/plot_base.py

```diff
@@ -80,6 +84,47 @@ def top_n(
             )
         return df
 
+    @staticmethod
+    def subsetted_top_n(df, top_n, facets=None):
```

Comment:
-Docstring added.
-Variable names changed, did I miss any bad names? (Me: check again)
-Used median instead. Since the plot highlights the median point, the ordering looks odd when weighted_average is used.
-Couldn't figure out a way to achieve ordering of best performing predictors with a LazyFrame. Also the ordering was not correctly implemented, it is fixed now.

[Functionality Change](https://github.com/pegasystems/pega-datascientist-tools/pull/69/commits/89ff3c2ddf631c68dd8989a9aa18613c6a437442)
[Minor naming consistency change](https://github.com/pegasystems/pega-datascientist-tools/pull/69/commits/a912216f5754c883ad6cbb1cfdf17b5e94640091)